Makefile

```diff
@@ -19,11 +19,17 @@ OUTPUT_DIR ?= _output
 
 TOOLS_DIR := hack/tools
 BIN_DIR := bin
-TOOLS_BIN_DIR := $(TOOLS_DIR)/$(BIN_DIR)
+TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/$(BIN_DIR))
 OUTPUT_BIN_DIR := $(OUTPUT_DIR)/$(BIN_DIR)
+GO_INSTALL = ./scripts/go_install.sh
```

Comment:
nit: maybe put this script in `hack/` instead of `scripts/`?

I know it doesn't entirely match the go standard project layout, but many k8s projects seem to use `hack` instead of `scripts`, and we already have some scripts in `hack`.